src/vanflow.c

```diff
@@ -770,7 +770,8 @@ static void _vflow_emit_record_as_log_TH(vflow_record_t *record)
     }
 
     record->never_logged = false;
-    qd_log(LOG_FLOW_LOG, QD_LOG_INFO, "%s", line);
+    // This log statement emits a single record as a log event
+    qd_log(LOG_FLOW_LOG, QD_LOG_DEBUG, "%s", line);
```

Comment:
I think this is the one you must not change.  This removes ALL flow-related content from the logs.